### PR TITLE
feat: implement reporter for htmlhint engine (#234)

### LINKED ISSUE #234: Implement reporter for htmlhint engine
### Reporter

Please implement a reporter file to the catalog for this engine

### PR DESCRIPTION
**Pull request by svoboda-rabstvo**
Closes #234

### CODE REVIEW

File: engine/htmlhint/reporter.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const template = module.exports = {
```
Comment (xferra) [inline engine/htmlhint/reporter.js]: As I already suggested: it seems to be a base implementation for most of the reporters. Why not to create npm package for that?

File: engine/htmlhint/reporter.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                        lineEnd: problem.line - 1,
+                        column: problem.col - 1,
+                        ruleId: problem.rule.id,
+                    };
+                }),
+            };
+        });
+    });
+
+    console.log(JSON.stringify(results));
```
Comment (xferra) [inline engine/htmlhint/reporter.js]: I also suggest to return an object from this method and perform conversion to JSON+console.write at least in the upper level.

File: engine/htmlhint/reporter.js (hunk 1 above)
Comment (svoboda-rabstvo) [reply engine/htmlhint/reporter.js]: but how use with `lhexec` ?

Comment (svoboda-rabstvo) [reply engine/htmlhint/reporter.js]: I propose this idea should be discussed in the separate issue

File: engine/htmlhint/reporter.js (hunk 2 above)
Comment (svoboda-rabstvo) [reply engine/htmlhint/reporter.js]: done

**Pull request merged**